docs/reference/support-policy.md

```diff
@@ -139,7 +139,7 @@ compatibility tables to describe the interoperability requirements of these piec
 
 #### Puppeteer
 
-| Stencil Version | Puppeteer v5-9 | Puppeteer v10 | Puppeteer v11-19 | Puppeteer v20 |
+| Stencil Version | Puppeteer v5-9 | Puppeteer v10 | Puppeteer v11-20 | Puppeteer v21 |
```

Comment:
the columns for 11-20 and 21 have the same emojis in them - should we consider just consolidating to a single column?

Reply:
yah - i'm ok with that 👍 updated